src/views/PVChargeConfig.vue

```diff
@@ -66,8 +66,8 @@
           </openwb-base-number-input>
           <hr />
           <openwb-base-alert :subtype="chargingSwitchRange < 1400 ? 'danger' : 'info'">
-            Die Differenzleistung zw. Ein- und Abschaltschwelle sollte mind. 1.4kW (230V x 6A) betragen. (Konfiguriert:
-            {{ chargingSwitchRange / 1000 }}&nbsp;kW)
+            Die Differenzleistung zw. Ein- und Abschaltschwelle sollte mind. 1,4 kW (230V x 6A) betragen. (Konfiguriert:
+            {{ (chargingSwitchRange / 1000).toLocaleString('de-DE') }}&nbsp;kW)
```

Comment:
Bitte keine hart codierten Regionseinstellungen verwenden. Mti `undefined` werden die Einstellungen des Browsers verwendet.
```suggestion
            {{ (chargingSwitchRange / 1000).toLocaleString(undefined) }}&nbsp;kW)
```